Update main README (#53)
capeprivacy/pycape: README.md

## Usage

Before running a function, you need to first get an access token (`<AUTH_TOKEN>`) with the [Cape CLI](https://github.com/capeprivacy/cli) by running `cape login`. Once logged into Cape, your token can be found in the `~/.config/cape/auth` file. The access token will be used when instantiating `Cape` client. If `access_token` attribute is None, it will try to load automatically the access token from your config file. Then you'll obtain a function id (`<FUNCTION_ID>`) and function hash (`<FUNCTION_HASH>`) once you have deployed a function with `cape deploy`.

### `run`

Run is used to invoke a function once with a single input. By default, inputs and outputs are expected to be bytes. 

> However you can easily serialize and deserialize inputs and outputs, using Serdio. To learn more about Serdio, you can find [this example](https://github.com/capeprivacy/pycape/tree/main/examples#mean-running-functions-on-python-types).

Reviewer: ```suggestion
However you can easily serialize and deserialize inputs and outputs, using Serdio. To learn more about Serdio, check out [this example](https://github.com/capeprivacy/pycape/tree/main/examples#mean-running-functions-on-python-types).
```

from pycape import Cape, FunctionRef

client = Cape(url="wss://enclave.capeprivacy.com")
client.run(

>     FunctionRef(function_id="<FUNCTION_ID>", function_hash="<FUNCTION_HASH>"),

Reviewer: I think we should replace these with fake hex strings. example:

```python
function_id = "ad134b923745c726"
function_hash = "1b5cb2a978697d6c5dadb876c8976adb"
f = FunctionRef(function_id, function_hash)
result = client.run(f, b"hello!")
print(result.decode())
>> hello!
```

Reviewer: same thing goes for the other example imo

>     "my_data".encode(),

Reviewer: ```suggestion
    b"my_data",
```

```py
from pycape import Cape

client = Cape(url="wss://enclave.capeprivacy.com")
client.connect(FunctionRef(function_id="<FUNCTION_ID>", function_hash="<FUNCTION_HASH>"))
client.invoke('my-data-1'.encode())
client.invoke('my-data-2'.encode())

> client.invoke('my-data-3'.encode())

Reviewer: ```suggestion
client.invoke(b"my-data-1")
client.invoke(b"my-data-2")
client.invoke(b"my-data-3")
```
